daft/expressions/expressions.py

```diff
@@ -2881,7 +2881,11 @@ def __repr__(self) -> str:
 class ExpressionImageNamespace(ExpressionNamespace):
     """Expression operations for image columns."""
 
-    def decode(self, on_error: Literal["raise"] | Literal["null"] = "raise") -> Expression:
+    def decode(
+        self,
+        on_error: Literal["raise"] | Literal["null"] = "raise",
+        mode: str | ImageMode | None = None,
+    ) -> Expression:
         """
         Decodes the binary data in this column into images.
```

Comment:
Let's add to this documentation too to explain the `mode` kwarg

I actually wonder if we should maybe default to setting it to `RGB` or something naive. it would help in the common case where a user might "just want it to work". WDYT?

Reply:
If a user wants the column to have multiple different modes, then they would have to explicitly pass in None to the argument, which feels a bit weird. I think it's fine this way.